lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -106,6 +106,7 @@
   overflow-y: auto;
   padding: var(--nx-spacing-6x) var(--nx-spacing-6x);
   white-space: normal;
+  z-index: 0;
```

Comment:
I think this should probably also be added to `.nx-page-sidebar` so that a dropdown in a sidebar doesn't have the same problem